### PR TITLE
UG, UI and HelpWindow Bug Fixes

### LINKED ISSUE #133: [PE-D][Tester A] PDF for UG is outdated
The UG PDF in the [release page ](https://github.com/AY2425S1-CS2103T-W11-1a/tp/releases) is wrong. It shows the user guide for AB3.

-------------
Labels: `type.DocumentationBug` `severity.High`
original: Feng1231/ped#1

### LINKED ISSUE #135: [PE-D][Tester B] Unupdated UG in release
The UG I downloaded from the release has its name updated to your project name but the content inside still belongs to the AB3. 

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: Potatoad88/ped#1

### LINKED ISSUE #137: [PE-D][Tester B] AB3 found in quick start
Should addressbook be changed to murphys list? Should the name of the jar file be changed to murphys_list.jar? (This was found under the quick start section of the UG)

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Potatoad88/ped#2

### LINKED ISSUE #138: [PE-D][Tester B] Mismatch between UG UI and actual product
The UI shown in the UG does not match the actual UI of the product when I run it. In your UG it is stated that "A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data." If this image of the UI in the UG is a mock up maybe you can specify that and remove the part that says "similat to the below".

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Potatoad88/ped#3

### LINKED ISSUE #139: [PE-D][Tester A] File name for data storage file is not intuitive
File name for data file is `AddressBook.json`, I would expect the name for data file to be `murphys_list`

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: Feng1231/ped#4

### LINKED ISSUE #142: [PE-D][Tester A] UI for patient information is not intuitive
* There is no label for each of the attribute of the user. 
* When I first see the `4` in the first contact for the sample data, it took me quite a while to figure out what that meant.
* Suggestion: describe what each attribute is in the UI

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: Feng1231/ped#7

### LINKED ISSUE #143: [PE-D][Tester A] Incorrect instructions in UG quick start
* UG says to run `java -jar addressbook.jar` but the jar file is named `murphys_list`
* High severity as target user is likely not good in CLI. A wrong instruction stated in the user guide will throw the reader off, and they will not be able to use the product.

-------------
Labels: `type.DocumentationBug` `severity.High`
original: Feng1231/ped#8

### LINKED ISSUE #144: [PE-D][Tester B] Examples in add command have issues
<i><video controls><source src="https://raw.githubusercontent.com/Potatoad88/ped/main/files/95b3ac0c-0a56-44bd-8b88-6bedc31564b3.mov" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/Potatoad88/ped/main/files/95b3ac0c-0a56-44bd-8b88-6bedc31564b3.mov</i>
Example of the add command throws an error when I use it in the app.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Potatoad88/ped#4

### LINKED ISSUE #145: [PE-D][Tester A] Wrong GUI example in UG
When reader reads the UG, he sees the following GUI:

Actual GUI 

Readers will be confused and wonder if they are using the correct applicaiton

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Feng1231/ped#9

### LINKED ISSUE #146: [PE-D][Tester A] Wrong example in UG quick start section
reproduce:
1. Copy the example in the quick start section for UG.

An error is thrown, when a command success is expected.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Feng1231/ped#10

### LINKED ISSUE #150: [PE-D][Tester A] View command's UI is missing triage information
`Triage` is an important attribute for the patients. When user types in `view s1234567a` and cannot view all the information of that patient, user would have to search for it manually, causing great inconvenience.

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: Feng1231/ped#11

### LINKED ISSUE #151: [PE-D][Tester A] Unable to resize the help window
The help window is too long for my computer. This might cause inconvenience for users with a slightly smaller screen.

suggestion: allow resize for help window

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: Feng1231/ped#12

### LINKED ISSUE #152: [PE-D][Tester A] website link in help window not updated
1. type `help` in the command line
1. locate the url of the user guide at the bottom

The url is not updated. Should be updated to be for murphy's list

Cause inconvenience, especially to new users, since they would require more guidance to use the product.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Feng1231/ped#13

### LINKED ISSUE #154: [PE-D][Tester A] List of command in help window is incomplete
Reproduce:
1. type `help` in the command line.

Commands like `log` and `schedule` is not stated in the help window.
Combined with the wrong user guide url provided, this is of high severity, as user would have no way to figure out that such commands existed, rendering the application not useful for them.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: Feng1231/ped#14

### LINKED ISSUE #158: [PE-D][Tester A] Wrong format provided in the UG for edit command
Conflicting format specified in UG and `edit` command's help message

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Feng1231/ped#16

### LINKED ISSUE #160: [PE-D][Tester A] Wrong format provided in the UG for Add command
Conflicting format specified in UG and `add` command's help message

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Feng1231/ped#17

### LINKED ISSUE #162: [PE-D][Tester A] First example provided in UG for add command is not correct
The first example in UG for add command provided is wrong.

Reproduce:
1. Copy first example in UG
1. paste into command line

error thrown when command success is expected.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Feng1231/ped#18

### LINKED ISSUE #165: [PE-D][Tester A] Second example provided in UG for add command is not correct
The second example in UG for add command provided is wrong.

Reproduce:
1. Copy second example in UG
1. paste into command line

error thrown stating Multiple values specified for t/, when command success is expected.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Feng1231/ped#19

### LINKED ISSUE #167: [PE-D][Tester A] Inconsistency in prefix meaning for various commands
For `add` and `edit`, `t/` means traige, `tag/` for tag. in `find` ,`t/` for finding tag?

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: Feng1231/ped#20

### LINKED ISSUE #171: [PE-D][Tester D] App Name in User Guide Needs Update
In the UG, the name of the app is still listed as 'AB3', which could be updated.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: publicmain/ped#7

### LINKED ISSUE #172: [PE-D][Tester A] Formatting error in UG
-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Feng1231/ped#22

### LINKED ISSUE #173: [PE-D][Tester D] Missing explanation in UserGuide
The Triage tag and appointment field is not explained in UserGuide or in app

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: publicmain/ped#8

### LINKED ISSUE #174: [PE-D][Tester C] UG Not updated at all
The UG was not up to date. Perhaps the team has uploaded the wrong UG?

-------------
Labels: `severity.High` `type.DocumentationBug`
original: jeanyang04/ped#1

### LINKED ISSUE #178: [PE-D][Tester D] Wrong documentation for Edit command in UG
The explanation of edi command in UG is wrong, in UG, it says the index must be a positive integer 1,2,3, but the error message in app shows Edits the details of the person identified by the patient's NRIC.

-------------
Labels: `type.DocumentationBug` `severity.High`
original: publicmain/ped#9

### LINKED ISSUE #184: [PE-D][Tester D] Missing Documentation in UG
In UG, there is no explanation of view command, but its already in App. 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: publicmain/ped#12

### LINKED ISSUE #186: [PE-D][Tester A] Missing glossary and documentation for triage not defined in user guide
Reproduce:
type `edit s1234567d t/j`

Expected:
the values and meaning for triage is well documented.

* User do not know that values of triage ranges from 1 to 5, until error thrown.
* User will not know what the value of 2 3 4 for triage stands for.
* Reader will also not know as it is not defined in the user guide

-------------
Labels: `type.DocumentationBug` `severity.High`
original: Feng1231/ped#25

### LINKED ISSUE #188: [PE-D][Tester B] Missing sort command
Sort command is in the UG but missing from the table of contents and command summary.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Potatoad88/ped#12

### LINKED ISSUE #192: [PE-D][Tester C] Help window does not expand to show all text when opened on smaller screens
On smaller screens, the help window is unscrollable and unable to resize, so some of the text at the bottom will be obstructed.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: jeanyang04/ped#8

### LINKED ISSUE #193: [PE-D][Tester C] Help page does not give the correct add command format
This is the add command format shown in help window. However, this format is incorrect as it is lacking the compulsory field triage "t/" and NRIC "i/".

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: jeanyang04/ped#9

### PR DESCRIPTION
**Pull request by Dominic-Khoo**
CHANGELOG:

- Updated UG and GH Home page to include missing commands, removed residual elements of AB3, updated with newer screenshots, images and accurate examples, removed GUI mismatch
- Rectified QuickStart instructions in UG
- Updated Help Window to include missing commands, updated URL link to UG, made window scrollable for ease of usage
- Updated Person Card UI to include patient attributes for clarity
- Updated UG and HelpWindow UI to include missing functionality of FindCommand (able to find patients through tags instead of just name)
- Updated ViewWindow UI to include triage and made formatting consistent
- Refactor file names e.g addressbook.json to murphyslist.json for better intuitiveness and clarity
- Updated JUnit test cases to match abovementioned functionality changes

Closes #167, closes #165, closes #162, closes #160, closes #158 closes #154 closes #152 closes #151 closes #150 closes #146 closes #145 closes #144 closes #143 closes #142 closes #139 closes #138 closes #137 closes #135 closes #133

### CODE REVIEW

File: src/main/java/seedu/address/model/UserPrefs.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 public class UserPrefs implements ReadOnlyUserPrefs {
 
     private GuiSettings guiSettings = new GuiSettings();
-    private Path addressBookFilePath = Paths.get("data" , "addressbook.json");
+    private Path addressBookFilePath = Paths.get("data" , "murphyslist.json");
```
Comment (itsnotsherm) [inline src/main/java/seedu/address/model/UserPrefs.java:17]: good catch

**Review by itsnotsherm — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -63,10 +69,14 @@ If you can type fast, Murphy's List can get your healthcare administrative tasks
...
-  e.g. `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g. `n/NAME [tag/TAG]` can be used as `n/John Doe tag/Parkinsons` or as `n/John Doe`.
+
+* Items with `…` after them can be used multiple times, including zero times.  
+  For example, `[tag/TAG]…` can be used as follows:
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+  - Not at all (i.e., ` ` )
+  - With one tag: `t/Parkinsons`
```
Comment (lim-jt) [inline docs/UserGuide.md]: t/ should be tag/ here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -63,10 +69,14 @@ If you can type fast, Murphy's List can get your healthcare administrative tasks
...
+  - With multiple tags: `t/Parkinsons t/Diabetic`
```
Comment (lim-jt) [inline docs/UserGuide.md]: same here, t/ should be tag/ here

File: docs/UserGuide.md (hunk 2 above)
Comment (Dominic-Khoo) [reply docs/UserGuide.md]: oops

**Pull request merged**